spring-cloud-gcp-pubsub/src/test/java/com/google/cloud/spring/pubsub/support/DefaultSubscriberFactoryTests.java

```diff
@@ -717,23 +717,6 @@ void testBuildGlobalSubscriberStubSettings_retryableCodes_pickConfiguration()
     assertThat(settings.pullSettings().getRetryableCodes()).containsExactly(Code.INTERNAL);
   }
 
-  @Test
-  void createSubscriberStubSucceeds_noSubscriptionNameAndNewConfiguration() {
```

Comment:
@JoeWang1127 you may have covered this already so apologies if this question is redundant! Any reason for removing this test case?

Reply:
Because the method under test, `DefaultSubscriberFactory.createSubscriberStub()`, is removed.

Reply:
Got it, thank you.